### PR TITLE
Add standalone build for 3.4 series

### PR DESCRIPTION
**Pull request by acoulton**
Merges the 3.3/develop branch up to 3.4/develop and updates the travis and composer configs for 3.4.x - continues kohana/kohana#50

### CODE REVIEW

File: tests/cache/FileTest.php [hunk 1]
```diff
@@ -5,6 +5,7 @@
  * @package    Kohana/Cache
  * @group      kohana
  * @group      kohana.cache
+ * @group      wip
```
Comment (enov) [inline tests/cache/FileTest.php:8]: Why this group? Is this added as a todo note?

**Comment by enov** [comment]
That `[ci skip]` on 2533c8d looks cool for sure :smile:

File: tests/cache/FileTest.php (hunk 1 above)
Comment (acoulton) [inline tests/cache/FileTest.php:8]: Drat, I must have put it in to filter the run when I was working on those tests within kohana/kohana. It's made it into 3.3 now as well :frowning: Probably best if I revert it separately and merge up again once this is in?

Comment (enov) [inline tests/cache/FileTest.php:8]: > Probably best if I revert it separately

Yeah, I'll merge now.

**Pull request merged**